src/Publisher/Tapatalk/TopicPublisher.php

```diff
@@ -5,7 +5,7 @@
  *
  * @package    release-notifier
  * @author     David Molineus <david.molineus@netzmacht.de>
```

Comment:
I use the coding standards of the Contao Community Alliance here, which requires that every contibutor add itself to the author list of the files (PHP) which got edited. Could you please add yourself here.

You might need to add the `.check-author.yml` if your configured git user isn't your real name. More information are available at https://github.com/phpcq/author-validation#configuration